docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -939,40 +939,10 @@ Code is below for reference
 Model 3 V&V for the relative risk with angina showed a lot of variability: 
     .. image:: Model3_VV_Angina.png
 
-  
-.. _uscvd4.7:
-
-4.7 Desired outputs
--------------------
-
-Outputs:
-
-#. Total population 
-#. Person-time 
-#. YLLs and YLDs
-#. Deaths 
-#. Transitions for each cause 
-#. Total exposure value * person time for all risk factors 
-#. Person time at or below target values for SBP and LDL-C 
-#. Healthcare appointments 
-#. Missed appointments 
-#. Person time on medication 
-#. Medication effect - exposure levels stratified by medication time 
```

Comment:
Is this the one you removed when you said via slack that you removed the "medication exposure?

Reply:
Yes. Basically I removed this whole list of bullets, but didn't add to the table anything about "medication exposure"